code/espurna/config/hardware.h

```diff
@@ -4299,6 +4299,26 @@
     #define RELAY3_TYPE             RELAY_TYPE_NORMAL
     #define RELAY4_TYPE             RELAY_TYPE_NORMAL
 
+// -----------------------------------------------------------------------------
+// ESP8266 Serial, example for use connected to a arduino
+//
+// -----------------------------------------------------------------------------
+
+#elif defined(ESP8266_SERIAL)
+
+    // Info
+    #define MANUFACTURER                "ESP8266"
+    #define DEVICE                      "SERIAL"
+
+    // Serial to MQTT
+    #define UART_MQTT_SUPPORT       1
+    #define UART_MQTT_USE_SOFT      0
+    #define UART_MQTT_HW_PORT       Serial
+    #define UART_MQTT_BAUDRATE      115200
+    #define TERMINAL_SUPPORT        0
```

Comment:
Looking at terminal.ino, it never calls Serial.read() unless debug_serial_support flag is on (which is off here).

Reply:
(this comment was meant to point to the disabled TERMINAL_SUPPORT)